modules/ROOT/pages/authentication.adoc

```diff
@@ -38,22 +38,53 @@ passwd:
     - name: jlebon
       ssh_authorized_keys:
         - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDC5QFS...
-        - sh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIIveEaMRW...
+        - ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIIveEaMRW...
     - name: miabbott
       ssh_authorized_keys:
         - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDTey7R...
 ----
 
+=== Using File References to SSH Keys
+
+Depending on the configuration variant and version you use, you can use file references to SSH public keys instead of inlining them.
```

Comment:
Would it make sense to s/file references/local file references/g?  "File references" could be referring to a file on the provisioned system, or a `storage.files` section, and the directives do include the word `local`.

Reply:
Fixed in [9436a55](https://github.com/coreos/fedora-coreos-docs/pull/514/commits/9436a55a77b48b3e11712d6e4b18137a33e7a0ca)

Reply:
Oh right – will fix that in a couple of minutes and squash the commits as well. Sorry about that – simply wanted to get this out of the way and didn't properly revisit the PR.